src/resolve/resolved-manager.c

```diff
@@ -1200,7 +1200,7 @@ int manager_compile_dns_servers(Manager *m, OrderedSet **dns) {
         return 0;
 }
 
-int manager_compile_search_domains(Manager *m, OrderedSet **domains, int filter_route) {
+int manager_compile_search_domains(Manager *m, OrderedSet **domains, bool filter_route) {
```

Comment:
Right, we only use it for building `search` in `resolv.conf`. If you want to keep the tri-state, how about switching to an enum? Otherwise, dropping the arg entirely WFM too.